Buildings/Fluid/DXSystems/Heating/AirSource/Validation/BaseClasses/PLRToPulse.mo

```diff
@@ -97,22 +104,24 @@ equation
     annotation (Line(points={{62,-10},{92,-10},{92,0},{120,0}},
       color={255,0,255}));
 
-annotation (Icon(
+annotation (
+  defaultComponentName="plrToPul",
+  Icon(
     coordinateSystem(preserveAspectRatio=false),
     graphics={Rectangle(extent={{-100,100},{100,-100}},lineColor={0,0,0},
       fillColor={255,255,255},fillPattern = FillPattern.Solid),
                                         Text(
         extent={{-160,140},{160,100}},
         textString="%name",
         textColor={0,0,255})}),
-  Diagram(defaultComponentName="plrToPul",
+  Diagram(
   coordinateSystem(preserveAspectRatio=false)),
   Documentation(info="<html>
 <p>
 This block calculates the time duration for which the DX coil needs to be kept enabled
 based on the part-load ratio input signal <code>uPLR</code> for the constant simulation
-run period <code>tPer</code>, and then generates an output enable signal 
-<code>yEna</code> for that duration. Once the component has been kept enabled for 
+run period <code>tPer</code>, and then generates an output enable signal
```

Comment:
@karthikeyad-pnnl It seems that in the simulation, the time limit should be `uPLR * tPer`?